server/requirements_common.txt

```diff
@@ -39,6 +39,7 @@ setuptools==69.0.2 ; python_version >= "3.9" and python_version < "3.13"
 tokenizers==0.13.3 ; python_version >= "3.9" and python_version < "3.13"
 tqdm==4.66.1 ; python_version >= "3.9" and python_version < "3.13"
 transformers==4.33.3 ; python_version >= "3.9" and python_version < "3.13"
+git+https://github.com/Saibo-creator/transformers-CFG.git@4d76433
```

Comment:
it would be preferred to fetch the library from pypi instead of github in the future do you know if this library has plan on pushing to pypi?